docs/starting-from-scratch.md

```diff
@@ -1,142 +1,449 @@
 
-If you’re struggling to decide between the two, then ask yourself the following: Do I want multiple completely separate bundles? If you don’t, I recommend Webpack because it has better docs, a larger community, and fewer gotchas. Otherwise, I’d go with JSPM, since Webpack has no plans to support dynamic runtime loading. 
+## Step One: Project Set up 
 
-Here is an example of how you can set up your [webpack.config](https://github.com/joeldenning/simple-single-spa-webpack-example/blob/master/webpack.config.js) file.
+Create a new directory to house our project, followed by a `src` folder to hold all of our micro-service applications.
 
-## Step Two: create a brand new html file
-The next step is to create what single-spa calls your “[root application](https://github.com/CanopyTax/single-spa/blob/master/docs/root-application.md).” Really your root application is just the stuff that initializes single-spa, and it starts with an html file.
+```bash
+mkdir single-spa-simple-example && cd single-spa-simple-example
+mkdir src
+```
+
+In the root of our new project, initialize the package manager of your choice and install single-spa. For this tutorial, we will be using [yarn](https://yarnpkg.com/en/). 
+
+```bash
+yarn init
+yarn add single-spa
+```
+
+
+### a) Set up Babel 
+In order to take advantage of the latest verion of Javascript, regardless of current browser compatability, we will use [Babel](https://babeljs.io/).
+
+Start by adding the following with your package manager: 
+
+<!-- SHOULD I Explain each of the following babel packages I am adding or just link to the docs? -->
+
+```bash
+yarn add babel-core babel-plugin-syntax-dynamic-import babel-preset-env babel-preset-latest babel-preset-react --dev
+```
+
+Then, in your root directory create a `.babelrc` file and include the following:
+
+```js
+// .babelrc
+{
+    "presets": [
+      ["env", {
+        "targets": {
+          "browsers": ["last 2 versions"]
+        }
+      }],
+      ["react"]
+    ],
+    "plugins": [
+      "syntax-dynamic-import"
+    ]
+  }
+```
+If you would like to learn more about what each of these things are doing, check out the [Babel docs](https://babeljs.io/docs/setup/). 
+
+### b) Set up webpack
+
+  *Once again, it is important to note that you do not have to use webpack in order to get up an running with single-spa. Checkout the documention on [Separating application](https://single-spa.js.org/docs/separating-applications.html) to learn more about the different ways you can use single-spa for your specific build.*
+
+  Start by adding webpack, webpack plugins and loaders. 
 
-### Example:
-``` html
-<!DOCTYPE html>
+*Webpack*
+```bash
+yarn add webpack webpack-dev-server webpack-cli --dev
+```
+
+*Plugins*
+```bash
+yarn add clean-webpack-plugin --dev
+```
+
+*Loaders*
+```bash
+yarn add style-loader css-loader html-loader --dev
+```
+
+  <!-- Should I walk through the purpose for each of the webpack plugins I am using? Or just link to the docs? -->
```

Comment:
I don't think so. Most people will just copy paste the commands. Documentation to webpack stuff is easily found with a google search, if they're interested, and it isn't particularly relevant to the single-spa tutorial